src/components/atoms/Card.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Card } from 'semantic-ui-react';
+
+const ExportCard = ({ children }) => {
+  return <StyledCard fluid>{children}</StyledCard>;
+};
+
+const StyledCard = styled(Card)`
```

Comment:
This looks like stuff theme overrides would be perfect for, since we're supposed to use this Card every now (at least I  assume that's the intention)

Reply:
Yes you might be right. I think we should use the theming for this, and create 'molecules' of our own instead of 'atoms' and handle the 'atoms' through theming. I still think it would be nice to have several type of Cards as 'molecules', for example a clickable card, a card with button, a listing card, etc.

Reply:
I agree!